src/RadixDlt.NetworkGateway.PostgresIntegration/CommonDbContext.cs

```diff
@@ -111,7 +111,7 @@ internal abstract class CommonDbContext : DbContext
 
     public DbSet<AccountDefaultDepositRuleHistory> AccountDefaultDepositRuleHistory => Set<AccountDefaultDepositRuleHistory>();
 
-    public DbSet<AccountResourcePreferenceRuleHistory> AccountDepositRuleHistory => Set<AccountResourcePreferenceRuleHistory>();
+    public DbSet<AccountResourcePreferenceRuleEntryHistory> AccountDepositRuleHistory => Set<AccountResourcePreferenceRuleEntryHistory>();
```

Comment:
Guess we should rename property (db set) name as well? `AccountDepositRuleHistory` -> `AccountResourcePreferenceRuleEntryHistory`:

Reply:
fixed.